root/static/scripts/release-editor/trackParser.js

```diff
@@ -323,6 +329,13 @@ releaseEditor.trackParser = {
             return data;
         }
 
+        //Use custom delimeter as separator.
+        if (options.useCustomDelimiter && options.customDelimiter != "") {
```

Comment:
`options.customDelimiter` is an observable, so this probably doesn't work as intended. You need to use `options.customDelimiter()` to unwrap the value.

There is a similar issue when setting the value below; it should be `options.customDelimiter(value)`, not `options.customDelimiter = value;`. However, this will change what the user entered in the field, which may be confusing. (Not an issue if you allow any RegExp syntax.)

Reply:
It actually seems to work as intended and not work if I use () in fact.